modules/analysis/rosbag-image-pipeline/image_dags/ros_image_pipeline.py

```diff
@@ -18,52 +18,63 @@
 import os
 import random
 import string
-import textwrap
+import time
 from datetime import timedelta
+from math import ceil
 from typing import TypeVar
 
 import boto3
 from airflow import DAG, settings
 from airflow.contrib.hooks.aws_hook import AwsHook
 from airflow.exceptions import AirflowException
 from airflow.models import Connection
+from airflow.operators.dummy_operator import DummyOperator
 from airflow.operators.python import PythonOperator
 from airflow.providers.amazon.aws.operators.batch import AwsBatchOperator
 from airflow.utils.dates import days_ago
+from airflow.utils.task_group import TaskGroup
 from boto3.dynamodb.conditions import Key
-from boto3.session import Session
-from mypy_boto3_batch.client import BatchClient
-from sagemaker import get_execution_role
 from sagemaker.processing import ProcessingInput, ProcessingOutput, Processor
 from sagemaker.pytorch.processing import PyTorchProcessor
 
 from image_dags import batch_creation_and_tracking
 from image_dags.dag_config import ADDF_MODULE_METADATA, DEPLOYMENT_NAME, MODULE_NAME, REGION
 
-# SET MODULE VARIABLES FOR IMAGE EXTRACTION
+# SET VARIABLES FOR EXTRACTION
 PROVIDER = "FARGATE"  # One of ON_DEMAND, SPOT, FARGATE
 FILE_SUFFIX = ".bag"
-VCPU = "4"
-MEMORY = "16384"
-CONTAINER_TIMEOUT = 300  # Seconds - must be at least 60 seconds
-IMAGE_TOPICS = ["/flir_adk/rgb_front_left/image_raw", "/flir_adk/rgb_front_right/image_raw"]
+IMAGE_TOPICS = [
+    "/flir_adk/rgb_front_left/image_raw",
```

Comment:
I would keep this here for now, because I think the pipeline will actually need to support ingestion of drives that have differing topics, so the topics to extract will be drive specific and calculated at runtime, and not dag specific. So I would leave it like this for now and refactor once we implement the ingest module which will pass on the list of image topics per drive to this pipeline